charts/rustrial-aws-eks-iam-auth-controller/values.yaml

```diff
@@ -29,6 +29,12 @@ podSecurityContext:
   {}
   # fsGroup: 2000
 
+env:
+  - name: RUST_LOG
+    value: debug
+  - name: KUBERNETES_SERVICE_HOST
```

Comment:
I am not sure whether setting `KUBERNETES_SERVICE_HOST` by default is the right thing to do. I am totally with you that we should provide a way (by means of the `env` value) to set it, but we should not set it by default.

What about adding something like this, where it is commented out, but still documented?

```yaml
env:
  # In case your are affected by one of the following bugs, you might need to explicitly set KUBERNETES_SERVICE_HOST.
  # - https://github.com/kube-rs/kube-rs/issues/153
  # - https://github.com/kube-rs/kube-rs/issues/587
  #
  #- name: KUBERNETES_SERVICE_HOST
  #  value: kubernetes.default.svc
```

Reply:
I like the idea & reasoning! The code was updated to move the `RUST_LOG` env variable back and just leave the `env` setting as an optional thing with a note. I chose not to include references to the bugs, as this project isn't affected directly (kube-rs version should be recent enough and rustls is not used). They were rather just a hint to the fix for me when working through their code.